filebeat/module/logstash/slowlog/_meta/fields.yml

```diff
@@ -12,10 +13,12 @@
       description: >
         The module or class where the event originate.
     - name: thread
+      # Keyword?
```

Comment:
👍

Reply:
Changed to `keyword` with `text` multi-field in https://github.com/elastic/beats/pull/10417.